includes/templates/responsive_classic/templates/tpl_modules_products_new_listing.php

```diff
@@ -36,19 +51,19 @@
         $display_products_name = '';
       }
 
-      if (PRODUCT_NEW_LIST_MODEL != '0' and zen_get_show_product_switch($products_new->fields['products_id'], 'model')) {
+      if (PRODUCT_NEW_LIST_MODEL != '0' and $flag_show_model) {
         $display_products_model = '<b>' . TEXT_PRODUCT_MODEL . '</b>' . $products_new->fields['products_model'] . str_repeat('<br>', substr(PRODUCT_NEW_LIST_MODEL, 3, 1));
       } else {
         $display_products_model = '';
       }
 
-      if (PRODUCT_NEW_LIST_WEIGHT != '0' and zen_get_show_product_switch($products_new->fields['products_id'], 'weight')) {
```

Comment:
as has been previously stated by you, most people do not update their in-use template files.  considering we are looking are replacing this method, show we not add a soon to deprecated call into the `zen_get_show_product_switch` method?

Reply:
I add changes like this to the Release Specific Upgrade Considerations page.  

https://docs.zen-cart.com/user/upgrading/release_specific_upgrade_considerations/